gdp/small_batch/gdp_small_batch.py

```diff
@@ -154,6 +154,8 @@ def coeff_init(m, k):
     def vol(m, i, j):
         """
         Volume Requirement for Stage j.
+        Equation:
+            m.v[j] >= log(m.s[i,j]) + m.b[i]
```

Comment:
You should avoid using m.XXYY because this is specific to the Pyomo implementation. Write the equation in latex format rather.

Reply:
write v[j] >= log(s[i,j]) + b[i]
This way for all the equations. Remove the m....